tests/parser/permit-generation-module.test.ts

```diff
@@ -51,41 +51,41 @@ jest.mock("@supabase/supabase-js", () => {
 
 // original rewards object before fees are applied
 const resultOriginal: Result = {
-  "user1": {
-    "total": 100,
-    "task": {
-      "reward": 90,
-      "multiplier": 1,
+  user1: {
+    total: 100,
+    task: {
+      reward: 90,
+      multiplier: 1,
     },
-    "userId": 1,
-    "comments": [
+    userId: 1,
+    comments: [
       {
-        "content": "comment 3",
-        "url": "https://github.com/user-org/test-repo/issues/57#issuecomment-2172704421",
-        "type": CommentType.COMMENTED,
-        "score": {
-          "reward": 10
-        }
-      }
-    ]
+        content: "comment 3",
+        url: "https://github.com/user-org/test-repo/issues/57#issuecomment-2172704421",
+        type: CommentType.COMMENTED,
+        score: {
+          reward: 10,
+        },
+      },
+    ],
   },
-  "user2": {
-    "total": 11.11,
-    "task": {
-      "reward": 9.99,
-      "multiplier": 1,
+  user2: {
+    total: 11.11,
+    task: {
+      reward: 9.99,
+      multiplier: 1,
     },
-    "userId": 1,
-    "comments": [
+    userId: 1,
+    comments: [
       {
-        "content": "comment 3",
-        "url": "https://github.com/user-org/test-repo/issues/57#issuecomment-2172704421",
-        "type": CommentType.COMMENTED,
-        "score": {
-          "reward": 1.12
-        }
-      }
-    ]
+        content: "comment 3",
+        url: "https://github.com/user-org/test-repo/issues/57#issuecomment-2172704421",
+        type: CommentType.COMMENTED,
+        score: {
+          reward: 1.12,
+        },
+      },
+    ],
   },
 };
```

Comment:
What's up with the formatter?

Reply:
I formatted it with prettier, I'm guessing it wasn't formatted before